\hypertarget{classdjango_1_1contrib_1_1gis_1_1geos_1_1error_1_1_g_e_o_s_exception}{}\doxysection{django.\+contrib.\+gis.\+geos.\+error.\+GEOSException Class Reference}
\label{classdjango_1_1contrib_1_1gis_1_1geos_1_1error_1_1_g_e_o_s_exception}\index{django.contrib.gis.geos.error.GEOSException@{django.contrib.gis.geos.error.GEOSException}}


Inheritance diagram for django.\+contrib.\+gis.\+geos.\+error.\+GEOSException\+:
% FIG 0


Collaboration diagram for django.\+contrib.\+gis.\+geos.\+error.\+GEOSException\+:
% FIG 1


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/django/contrib/gis/geos/error.\+py\end{DoxyCompactItemize}
